frontend/src/Products/connector.js

```diff
@@ -16,10 +19,14 @@ import { getProductsResult } from '../selectors';
  * @return {Object} The extended component props.
  */
 const mapStateToProps = (state, props) => ({
-  products: getProductsResult(state, props.settings.queryType, {
+  ...getProductsResult(state, props.settings.queryType, {
     sort: props.settings.sortOrder,
     value: props.settings.queryParams,
-  }).products,
+  }, props.id),
+  isFetching: getProductsFetchingState(state, props.settings.queryType, {
```

Comment:
You should see if you can simplify this. Both selectors use exactly the same parameters ...